### DISCUSSION TITLE
Roadmap Project

### DISCUSSION THREAD

**Discussion by Cahllagerfeld** [EddieHubCommunity/EddieHubLive#54] [Ideas]
## Context
This project is currently floaing around a bit, with an unclear direction. We should change that.

## Goal
The purpose of this RFC is to come up with ideas which features might be added to this project
There are a few things to consider:
- accessibility
- map

### Used technologies
Short overview over proposed technologies
#### Tailwind
For being consistent across (most) Eddiehub projects, we should use tailwind for the implementation. By doing so we can setup our own design system for consistency, and also have the option to add a dark theme without huge effort.

#### leaflet
This is the map that is currently used. By keeping it we wouldn't have to rewrite the entire display logic

### UI Decisions
- The layout is currently split into 50% events and 50% map. We can keep this, for now, I think the map wouldn't need to be that big though. In my opinion, something for follow-up. An option here would be turning it into a dashboard-like app, displaying some other metrics and KPIs as well on the dashboard. (Overall Events, different Contributors, ...)
- Especially the orange boxes that are displayed when a new event is displayed are much too vibrant.
In my opinion, we should add the new events as a card and use orange as an accent color somewhere on the card.

### Features
- Add more of a dashboard-style to this project, to not have the events as live events, but also other metrics

## Open Questions
- Change the entire layout from a half-half to a different one?
- Stick with the proposed technologies

## Next Steps

- [ ] collect and discuss ideas
- [ ] turn things to issues and start implementing 🎉

**Comment by amandamartin-dev** [comment]
As far as the tech, I'm a huge fan of leaflet and it's easy to customize if we ever want to do anything more with map widgets or marker popups, etc. Same feeling for tailwind, it's very approachable to use and like you mentioned is used across other projects so easy to pop around for contributors. So my vote would be to stay with the existing tech unless a compelling reason comes up to change.

For your question about map vs event real estate, what is the main purpose of this app? And who is the user? I think that will help me brainstorm a bit.  
Another aside is to think about what the mobile vs desktop view should look like especially if we add more dashboard like elements.

**Reply by eddiejaoude** [reply]
For the design, I bought the full tailwindui prebuilt components, so we can use those and then customise further when required.

The initial goal of the project was to encourage people to contribute while I am live streaming and it can show in realtime on the map their contribution - I am always keen to get people who are watching the live stream involved in contributions

**Reply by amandamartin-dev** [reply]
Ah! Yes, you mentioned this before.   I will have to join a live stream next time to see it in action, but definitely ideas that will be rewarding to those that are tuned in similar to issue #21 

If the data is available, could add some kind of badging or other indicator for various things that happen during the stream: new to the community, first issue, ...  I'm sure there's more, I will keep an eye out for the next one to tune in for more ideas.
